### DISCUSSION TITLE
How to detect if a user is using temporary email service?

### DISCUSSION THREAD

**Discussion by montasim** [STRINGLABITSOLUTIONS/Bhojon#301] [Q&A] [answered]
So far I have tried this API: https://www.disify.com/api/email/your@email.com which is not reliable. I need a more accurate solution and do not want to pay for this.

**Comment by fcasamento** [comment, marked as answer]
@montasim, hello. See if any of these services works for you.
https://block-temporary-email.com/
or
https://verifymail.io/disposable-email

**Reply by montasim** [reply]
@fcasamento, thanks for your suggestions. I will let you know it it works.
